dm/pkg/shardddl/optimism/lock.go

```diff
@@ -307,7 +307,7 @@ func (l *Lock) TrySync(info Info, tts []TargetTable) (newDDLs []string, cols []s
 		// special case: check whether DDLs making the schema become part of larger and another part of smaller.
 		if _, err = prevTable.Compare(nextTable); err != nil {
 			return emptyDDLs, emptyCols, terror.ErrShardDDLOptimismTrySyncFail.Delegate(
-				err, l.ID, fmt.Sprintf("there will be conflicts if DDLs %s are applied to the downstream. old table info: %s, new table info: %s", ddls, prevTable, nextTable))
+				err, l.ID, fmt.Sprintf("there will be conflicts if DDLs are applied to the downstream. old table info: %s, new table info: %s", prevTable, nextTable))
```

Comment:
why remove ddls from log? 🤔

Reply:
Because https://github.com/pingcap/tiflow/blob/master/dm/pkg/terror/error_list.go#L1085 the error in dm-worker already has ddls, I want to remove one of them.